### DISCUSSION TITLE
Jetstream change Team model directory

### DISCUSSION THREAD

**Discussion by ahafizi** [laravel/framework#35756] [Q&A] [answered]
Hi, I have some problem with Jetstream Inertia after I move Team, Membership and User model to another directory. And I try view the {host}/user/profile url I take 

```Class 'App\Models\Team' not found``` 

How could I solve it?

**Comment by ahafizi** [comment, marked as answer]
Hooray! Solved I was find.

in **JetstreamServiceProvider** method boot override the 
```Laravel\Jetstream\Jetstream::teamModel = Team::class;```

**Comment by osbre** [comment]
I suggest using the provided `use` methods rather than diving into the "guts" and manually assigning the static properties.
```php
Jetstream::useTeamModel(Team::class);
Jetstream::useUserModel(User::class);
Jetstream::useMembershipModel(Membership::class);
Jetstream::useTeamInvitationModel(TeamInvitation::class);
```
